app/assets/stylesheets/header.scss

```diff
@@ -0,0 +1,38 @@
+@import "variables";
+.authentication {
```

Comment:
you could space out classes a little bit and add a newline after `@import` also, it will be easier to read